UndertaleModTool/Scripts/Community Scripts/TouchControls_data/gml_Object_obj_mobilecontrols_Draw_64.gml

```diff
@@ -1,22 +1,24 @@
-draw_sprite_ext(spr_black, 0, 0, 0, 1, 1, 0, c_white, black_fade)
+var coof = display_get_gui_width() / 640
+var coofH = display_get_gui_height() / 480
```

Comment:
please either comment what "coofH" mean, or user different variable names

Reply:
coofH - ratio of scale for horizontal axis, i think this one is useless, because im using scaling based on 640x480 gui resolution. (sorry, in my language it was like coof, but i forgot to google this, i will change it now.)

Reply:
ok, horizontal one was very helpful. i did latest test, and now i will send it. (also, i fixed ratio for config)